Sources/Build/BuildOperationBuildSystemDelegateHandler.swift

```diff
@@ -806,8 +821,9 @@ final class BuildOperationBuildSystemDelegateHandler: LLBuildBuildSystemDelegate
         queue.sync {
             self.progressAnimation.complete(success: success)
             if success {
+                let message = cancelled ? "Build cancelled!" : "Build complete!"
```

Comment:
Should this say "build failed" if it failed?

Reply:
Yah, we could do that. I think in most cases it'll be redundant as there's the underlying error right above?

Reply:
Wait, "Build completed!" should only be shown on success. Cancellation is treated as success (which I agree is confusing), but on failed builds there should be no message at all right now. Are you seeing something different?

Reply:
Oh right — you're totally correct.  I was thinking of the `xcodebuild` output.  I'm not seeing anything different, so this looks good to me.  Sorry for the confusion.